papers/david_dotson/david_dotson.rst

```diff
@@ -324,18 +324,19 @@ We can add categories to individual Treants:
     # for all members
     >>> b.categories.add({'plant': 'tree'})
 
-We can access categories for an individual member:
+and we can access categories for individual Treants:
```

Comment:
So, keep the change or revert? From what you said it sounds like an endorsement, but not sure. Need clarification.